README.md

```diff
@@ -16,7 +16,8 @@ IoT Platform service will also be able to serve the flight information to all th
 ## Architecture
 Following figure shows the high-level architecture of a Cloud-based Air Traffic Control that relies on cheap Ground Stations to track flights
 
-![alt tag](https://github.com/IBM/air-traffic-control/blob/master/assets/architecture_diagram_v2.png)
+![alt tag](https://github.com/ljbennett62/air-traffic-control/blob/master/assets/arch-iot-airtrafficcontrol-1024x878.png)
+![alt tag](https://github.com/ljbennett62/air-traffic-control/blob/master/assets/AR-steps.png)
```

Comment:
The explanation of steps in `AR-steps.png` is way off. It would be better to have a meeting to talk about these steps so that they can be accurate.